Makefile

```diff
@@ -506,16 +511,21 @@ define wait-for-ingress
 endef
 
 define generate
-	$(call controller-gen,object:headerFile="hack/boilerplate.go.txt" paths="./...")
+	$(call install-controller-gen)
+	@echo -e "\n$(YELLOW)Generating the API code$(NC)"
+	$(GOBIN)/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
 endef
 
-define manifest
-	$(call controller-gen,crd:crdVersions=v1 paths="./..." output:crd:artifacts:config=chart/k8gb/templates/crds)
+define crd-manifest
+	$(call install-controller-gen)
+	@echo -e "\n$(YELLOW)Generating the CRD manifests$(NC)"
+	@echo -n "{{- if .Values.k8gb.deployCrds }}" > $(GSLB_CRD_YAML)
+	$(GOBIN)/controller-gen crd:crdVersions=v1 paths="./..." output:crd:stdout >> $(GSLB_CRD_YAML)
+	@echo "{{- end }}"  >> $(GSLB_CRD_YAML)
```

Comment:
looks like it will tamper the git status?

Reply:
@ytsarev not sure I'm following

Reply:
@somaritane i've meant from the PoV that it will affect `git status` after regeneration but it is actually valid in case of API changes. And if there are no API changes in the working tree the GSLB_CRD_YAML file will stay intact. So all good, approving